src/main/java/com/tstepnik/planner/service/TaskService.java

```diff
@@ -0,0 +1,63 @@
+package com.tstepnik.planner.service;
+
+import com.tstepnik.planner.domain.Importance;
+import com.tstepnik.planner.domain.Task;
+import com.tstepnik.planner.repository.TaskRepository;
+import com.tstepnik.planner.security.auth.CustomUserDetails;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.stereotype.Service;
+
+import java.security.Principal;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TaskService {
+
+    private final Importance DEFAULT_IMPORTANCE = Importance.INCIDENTAL;
+
+    private final TaskRepository taskRepository;
+
+    public TaskService(TaskRepository taskRepository) {
+        this.taskRepository = taskRepository;
+    }
+
+    public List<Task> findAll() {
+        return taskRepository.findAll();
+    }
+    public Optional<Task> getTask(Long id){
+        return taskRepository.findById(id);
+    }
+
+    public Task addTask(Task task, Long userId) {
+        task.setImportance(DEFAULT_IMPORTANCE);
```

Comment:
Why are you adding a task always with default importance?

Reply:
Because this is one of the assumption of the projects. Task should have importance, it motivate User to finish his tasks or tell him which he should finish at first or postpone if he don't have time.

Reply:
Because it's one of the assumptions of the project. It's good to have direction which task is more important and which is not. User can change the default but if he don't that means task is not enough important for him.